LibraBFT/ImplShared/Util/RWST.agda

```diff
@@ -135,13 +143,37 @@ module LibraBFT.ImplShared.Util.RWST (ℓ-State : Level) where
        where open RWST-do
 
   -- Derived Functionality
-
-  maybeMP : RWST Ev Wr St (Maybe A) → B → (A → RWST Ev Wr St B)
+  maybeSM : RWST Ev Wr St (Maybe A) → RWST Ev Wr St B → (A → RWST Ev Wr St B) → RWST Ev Wr St B
```

Comment:
Oh, I see I was wrong about replacing these with standard library functions.  I agree with @haroldcarr that it's better to keep these definitions in the same place as the Haskell code.